build.gradle

```diff
@@ -22,95 +22,27 @@ ext {
 ext {
     // libs
     automatticAboutVersion = '1.4.0'
-    automatticRestVersion = '1.0.8'
     automatticTracksVersion = '5.1.0'
     gutenbergMobileVersion = 'v1.120.1'
     wordPressAztecVersion = 'v2.1.3'
     wordPressFluxCVersion = 'trunk-79f2bc35285748c715e00f2a6648ed6831632178'
     wordPressLoginVersion = '1.16.0'
-    wordPressPersistentEditTextVersion = '1.0.2'
     wordPressUtilsVersion = '3.14.0'
-    indexosMediaForMobileVersion = '43a9026f0973a2f0a74fa813132f6a16f7499c3a'
     gravatarVersion = '1.0.0'
 
-    // debug
-    flipperVersion = '0.245.0'
-    soLoaderVersion = '0.10.5'
-
     // main
-    androidInstallReferrerVersion = '2.2'
-    androidVolleyVersion = '1.2.1'
     androidxAnnotationVersion = '1.6.0'
-    androidxAppcompatVersion = '1.6.1'
-    androidxArchCoreVersion = '2.2.0'
-    androidxCameraVersion = '1.3.4'
     androidxComposeBomVersion = '2023.10.00'
     androidxComposeCompilerVersion = '1.5.9'
-    androidxComposeNavigationVersion = '2.7.6'
-    androidxCardviewVersion = '1.0.0'
-    androidxConstraintlayoutVersion = '2.1.4'
-    androidxConstraintlayoutComposeVersion = '1.0.1'
-    androidxCoreVersion = '1.10.0'
-    androidxActivityVersion = '1.8.0'
-    androidxFragmentVersion = '1.6.2'
-    androidxGridlayoutVersion = '1.0.0'
-    androidxLifecycleVersion = '2.6.2'
-    androidxPercentlayoutVersion = '1.0.0'
-    androidxPreferenceVersion = '1.2.0'
-    androidxRecyclerviewVersion = '1.3.0'
-    androidxSwipeToRefreshVersion = '1.1.0'
-    androidxViewpager2Version = '1.0.0'
-    androidxWorkManagerVersion = "2.9.0"
-    androidxWebkitVersion =  '1.11.0'
-    androidxComposeMaterial3Version = '1.1.1'
-    apacheCommonsTextVersion = '1.10.0'
-    coilComposeVersion = '2.4.0'
-    chrisbanesPhotoviewVersion = '2.3.0'
-    eventBusVersion = '3.3.1'
-    facebookShimmerVersion = '0.5.0'
+    androidxGridlayoutVersion = '1.0.0' // unused needs verify?
     firebaseBomVersion = '32.1.0'
-    firebaseIidVersion = '21.1.0'
-    glideVersion = '4.16.0'
-    googleAutoServiceVersion = '1.0.1'
-    googleExoPlayerVersion = '2.13.3'
-    googleFlexboxlayoutVersion = '3.0.0'
-    googleGsonVersion = '2.10.1'
-    googleMaterialVersion = '1.9.0'
-
-    googleMLKitBarcodeScanningVersion = '17.2.0'
-    googleMLKitBarcodeScanningCommonVersion = '17.0.0'
-    googleMLKitTextRecognitionVersion = '16.0.0'
-    googlePlayReviewVersion = '2.0.1'
-    googlePlayServicesAuthVersion = '20.4.1'
-    googlePlayServicesCodeScannerVersion = '16.0.0-beta3'
-    jsoupVersion = '1.16.2'
-    kotlinxCoroutinesVersion = '1.7.3'
-    lottieVersion = '6.1.0'
-    philjayMpAndroidChartVersion = 'v3.1.0'
-    squareupKotlinPoetVersion = '1.16.0'
-    squareupRetrofitVersion = '2.9.0'
+    googleAutoServiceVersion = '1.0.1' // unused needs verify?
+    googlePlayServicesCodeScannerVersion = '16.0.0-beta3' // unused needs verify?
```

Comment:
same

Reply:
👋 @neeldoshii and yes, please feel free to remove any such version that is unused ([same](https://github.com/wordpress-mobile/WordPress-Android/pull/21029#discussion_r1664291077))!

FYI: For these particular versions above I am seeing the dependencies being removed but not the version declaration itself, which was just an overside. Thus, those versions can be indeed safely removed:
- googleAutoServiceVersion -> 53a4dd0c2b50499272193649a3e9832f51db7b2d
- googlePlayServicesCodeScannerVersion -> 4e9d7e95efe438fb1281d9aa654a1ce7d86f1ed3 -> 8c18f4d831250c5a60fd3f9d4ad569a6cb418537 -> 260f8ed2ed04126678fcdb70d3a30e89c218d399